project/README.md

```diff
@@ -102,6 +102,7 @@ Every project is unique, and we provide a variety of options to ensure that your
 | `language_code`       | Language to be used on the site.                                                                                                                     | **pt-br**                     |
 | `github_organization` | Used for GitHub, GitLab, and Docker repositories.                                                                                                             | **collective**                |
 | `container_registry`  | Container registry to be used.                                                                                                                       | **github**                    |
+| `storage`             | Storage backend to be used.                                                                                                                          | **github**                    |
```

Comment:
I don't think `github` is a correct example storage option.

Reply:
That's true; it's what I got for c&p...